libafl/src/executors/forkserver.rs

```diff
@@ -644,6 +644,7 @@ impl<'a> ForkserverExecutorBuilder<'a, StdShMemProvider> {
     }
 
     /// The harness
+    #[must_use]
```

Comment:
In the command executor this is called `program`, what do you prefer? I don't mind either way but it should be consistent. Think I used program since it mimics the rust docs somewhat